.github/workflows/Dockerfile

```diff
@@ -0,0 +1,30 @@
+# Install rust for Noname
+FROM rust as noname-installer
+
+# Install Noname CLI
+RUN cargo install --git https://www.github.com/zksecurity/noname
```

Comment:
The CLI is better to build from the current branch, otherwise this doesn't test against the latest code.

I think you can add a COPY instruction to include the noname repo into the docker image, so the CLI can be built and installed from source code.